UNITY-8128: adapt doc for new label (#1278)
ePages-de/epages-docs: apps/data-types.md

| sfUrl | string | The link to the storefront URL of the product. |
| productNumber | string | The product number. |
| isProductNumberVisible | boolean | Indicates if the product number is displayed in the storefront. |

> | isNew | boolean | Indicates if the product should be displayed as in the storefront. |

Reviewer: I would suggest to slightly change the wording. Furthermore I added the "word" as it was missing.
```suggestion
| isNew | boolean | Indicates if the product is displayed as new in the storefront. |
```

Author: @dkluever22 you are totally right it must also be adapted in '_raml/apps/search/post_search_products_response.json'

| sfUrl | string | The link to storefront URL of the product. |
| productNumber | string | The product number. |
| isProductNumberVisible | boolean | Indicates if the product number is displayed in the storefront. |

> | isNew | boolean | Indicates if the product should be displayed as new in the storefront. |

Reviewer: I would suggest a slightly different wording:
```suggestion
| isNew | boolean | Indicates if the product is displayed as new in the storefront. |
```
